Nordic nrf52810_pca10040 is missing default bluetooth configuration options

When building sample applications with Bluetooth using the nrf52810_pca10040 board configuration, an error is emitted:

    C:/zephyr/drivers/bluetooth/hci/h4.c: In function '_bt_uart_init':
    C:/zephyr/drivers/bluetooth/hci/h4.c:462:30: error: 'CONFIG_BT_UART_ON_DEV_NAME' undeclared (first use in this function);
    did you mean 'CONFIG_UART_0_NAME'?
      h4_dev = device_get_binding(CONFIG_BT_UART_ON_DEV_NAME);
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~~
                                  CONFIG_UART_0_NAME
    C:/zephyr/drivers/bluetooth/hci/h4.c:462:30: note: each undeclared identifier is reported only once for each function it appears in
    [34/128] Building C object zephyr/CMak....dir/drivers/timer/nrf_rtc_timer.c.obj
    ninja: build stopped: subcommand failed.

Selecting the option Bluetooth -> Bluetooth Controller fixes this issue but it should be added to the default configuration as it is with the nrf52840_pca10056 board.

--- a/boards/arm/nrf52810_pca10040/Kconfig
+++ b/boards/arm/nrf52810_pca10040/Kconfig
@@ -11,4 +11,9 @@ config BOARD_ENABLE_DCDC
         select SOC_DCDC_NRF52X
         default y
 
+# BT_CTLR depends on BT. When BT is enabled we should default to also
+# enabling the controller.
+config BT_CTLR
+	default y if BT
+
 endif # BOARD_NRF52810_PCA10040

